docs/_Docs_Transactions.md

```diff
@@ -4,78 +4,86 @@ section: Docs
 order: 5
 ---
 
-Interacting with the Ethereum blockchain in a way which will cause its state to change is done with a transaction. In Tailor we have various types of `Transaction` which handle everything for you surrounding sending a transaction.
+Interacting with the Ethereum blockchain in a way which will cause its state to change is done with a transaction. In Tailor we have various types of transactions, which handles everything for you when sending a transaction.
 
 ## Contract Transactions
 
-The `contract` transaction is the one which you'll probably use most of the time. If you do not specify a different type of transaction in the method overrides for your Tailor instance, the contract type will be used as a default for all methods.
+The `contract` transaction is the one which you'll probably use most of the time. If you do not specify a different type of transaction in the method overrides for your Tailor instance, the `contract` type will be used as a default for all methods.
 
 It automatically encodes transaction data based on function parameters passed to it, and sets the `to` address to the `contractAddress` of the Tailor instance.
 
 ```js
+
 const client = await Tailor.load({
   methods: {
     myMethod: {
-      type: 'contract'
-    }
-  }
-})
+      type: 'contract',
+    },
+  },
+});
 
 await client.methods.myMethod({
   myParam: 'a string',
-  otherParam: 42
-}).send()
+  otherParam: 42,
+}).send();
+
 ```
 
-## MultiSig Transactions
+## Multisignature Transactions
 
-These are for use with contract methods which require a transaction to be signed by multiple parties, specifically designed around [`simple-multisig`](https://github.com/christianlundkvist/simple-multisig). See [Multisig](https://docs.colony.io/tailor/docs-multisignature-transactions/) docs for more in depth explanation.
+The `multisig` transactions is used with contract methods which require a transaction to be signed by multiple parties. The `multisig` transaction was designed around [`simple-multisig`](https://github.com/christianlundkvist/simple-multisig). See [Multisignature Transactions](https://docs.colony.io/tailor/docs-multisignature-transactions/) in the [colonyJS docs](https://docs.colony.io/colonyjs/docs-overview/) for more information.
 
 ```js
+
 const client = await Tailor.load({
   methods: {
     myMethod: {
```

Comment:
Ah my mistake, the `type` should actually be like this (with the options moved here):

```js
type: {
  name: 'multisig',
  options: {
    getRequiredSigners,
    multiSigFunctionName,
    getMultiSigNonce
  }
}
```

Reply:
👍